src/instruments/src/SD/Pages/Hyd/Hyd.tsx

```diff
@@ -209,243 +123,329 @@ const RAT = ({ x, y }: RATProps) => {
-            <line className={!pumpDetectLowPressure ? 'GreenLine' : 'Hide'} x1={x} y1={y} x2={x} y2={y - 80} />
-            <line className={pumpOn ? 'Hide' : 'AmberLine'} x1={x - 12} y1={y - 64} x2={x + 12} y2={y - 64} />
-            <line className={hydLevelLow || !fireValve ? 'AmberLine' : 'GreenLine'} x1={x} y1={y} x2={x} y2={y - 48} />
-            <text
-                id="ELEC-centre"
-                className={
-                    pumpDetectLowPressure && pumpOn ? 'RatPtuElec FillAmber' : 'Hide'
-                }
-                x={x}
-                y={y - 64}
-                alignmentBaseline="central"
-            >
-                LO
-
-            </text>
-
-        </>
+        <SvgGroup x={x} y={y}>
+            <line className={pumpSwitchLowPressure ? 'AmberLine' : 'GreenLine'} x1={0} y1={-42} x2={0} y2={pumpLineYUpper} />
+            <rect className={pumpSwitchLowPressure || !pumpOn ? 'AmberLine' : 'GreenLine'} x={-21} y={-42} width={42} height={42} />
+            <line className={!pumpSwitchLowPressure && pumpOn ? 'GreenLine' : 'Hide'} x1={0} y1={-1} x2={0} y2={-41} />
+            <line className={pumpOn ? 'Hide' : 'AmberLine'} x1={-12} y1={-21} x2={12} y2={-21} />
+            <text className={pumpSwitchLowPressure && pumpOn ? 'Standard Amber' : 'Hide'} x={-13} y={-14}>LO</text>
+        </SvgGroup>
     );
 };
 
 type HydEngValveProps = {
-    system: string,
     x: number,
     y: number,
-    fireValve: number,
-    hydLevelLow: boolean
+    fireValve: boolean,
+    lowLevel: boolean
 }
 
-const HydEngValve = ({ system, x, y, fireValve, hydLevelLow } : HydEngValveProps) => {
-    if (system === 'BLUE') {
-        return (
-            <line className={!hydLevelLow ? 'GreenLine' : 'AmberLine'} x1={x} y1={y + 33} x2={x} y2={y} />
-        );
-    }
-
-    return (
-        <>
-            <circle className={fireValve ? 'GreenLine' : 'AmberLine'} cx={x} cy={y + 16} r="16" />
-            <line className={fireValve ? 'GreenLine' : 'Hide'} x1={x} y1={y + 32} x2={x} y2={y} />
-            <line className={fireValve ? 'Hide' : 'AmberLine'} x1={x - 10} y1={y + 16} x2={x + 10} y2={y + 16} />
-        </>
-    );
-};
+const HydEngValve = ({ x, y, fireValve, lowLevel } : HydEngValveProps) => (
+    <SvgGroup x={x} y={y}>
+        <line className={fireValve && !lowLevel ? 'GreenLine' : 'AmberLine'} x1={-0} y1={0} x2={0} y2={-68} />
+        <circle className={fireValve ? 'GreenLine' : 'AmberLine'} cx={0} cy={21} r="21" />
+        <line className={fireValve ? 'GreenLine' : 'Hide'} x1={0} y1={42} x2={x} y2={0} />
+        <line className={fireValve ? 'Hide' : 'AmberLine'} x1={-21} y1={21} x2={21} y2={21} />
+    </SvgGroup>
+);
+
+const levels = [
+    { system: 'GREEN', max: 14.5, low: 3.5, norm: 2.6 },
+    { system: 'BLUE', max: 6.5, low: 2.4, norm: 1.6 },
+    { system: 'YELLOW', max: 12.5, low: 3.5, norm: 2.6 },
+];
```

Comment:
How about using something like this? We can avoid the `.filter` down the line.
```suggestion
const levels = {
    'GREEN': { max: 14.5, low: 3.5, norm: 2.6 },
    'BLUE': { max: 6.5, low: 2.4, norm: 1.6 },
    'YELLOW': { max: 12.5, low: 3.5, norm: 2.6 },
};
```